index.tsx

```diff
@@ -973,7 +1008,7 @@ export function compiler(
       _inline = SHOULD_RENDER_AS_BLOCK_R.test(input) === false
     }
 
-    const arr = emitter(
+    let arr = emitter(
```

Comment:
With the changes below, `arr` can remain a const value.

Reply:
330f0440d4aa981e3c6652245ca0b078811b48e1